commands.js

```diff
@@ -2192,6 +2192,7 @@ exports.commands = {
 		if (!this.can('forcerename', targetUser)) return false;
 
 		this.addModCommand("" + targetUser.name + " was namelocked by " + user.name + "." + (reason ? " (" + reason + ")" : ""));
+		room.unlinkHide(targetUser, 'NAMELOCK');
```

Comment:
Sidenote: I'm not sure why namelocks didn't do this before...

With that in mind, in the case that it wasn't forgotten, if this was intentional, then please let me know!